Integrations/GoogleCloudStorage/GoogleCloudStorage.yml

```diff
@@ -0,0 +1,515 @@
+- display: Use system proxy settings
+  name: use_proxy
+  required: false
+  type: 8
+description: Google Cloud Storage is a RESTful online file storage web service for
+  storing and accessing data on Google Cloud Platform infrastructure.
+display: Google Cloud Storage
+name: Google Cloud Storage
+script:
+  commands:
+  - deprecated: false
+    description: Retrieves the list of buckets in the project
+    execution: false
+    name: gcs-list-buckets
+    outputs:
+    - contextPath: GCS.Bucket.Name
+      description: Bucket name (which is also its ID)
+      type: String
+    - contextPath: GCS.Bucket.TimeCreated
+      description: Bucket creation time
+      type: Date
+    - contextPath: GCS.Bucket.TimeUpdated
+      description: Last bucket modification time
+      type: Date
+    - contextPath: GCS.Bucket.OwnerID
+      description: Bucket owner ID
+      type: String
+  - arguments:
+    - default: true
+      description: Name of the bucket to retrieve
+      isArray: false
+      name: bucket_name
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieves bucket information
+    execution: false
+    name: gcs-get-bucket
+    outputs:
+    - contextPath: GCS.Bucket.Name
+      description: Bucket name (which is also its ID)
+      type: String
+    - contextPath: GCS.Bucket.TimeCreated
+      description: Bucket creation time
+      type: Date
+    - contextPath: GCS.Bucket.TimeUpdated
+      description: Last bucket modification time
+      type: Date
+    - contextPath: GCS.Bucket.OwnerID
+      description: Bucket owner ID
+      type: String
+  - arguments:
+    - default: true
+      description: Name of the bucket to create
+      isArray: false
+      name: bucket_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Access controls for the bucket
+      isArray: false
+      name: bucket_acl
+      predefined:
+      - authenticatedRead
+      - private
+      - projectPrivate
+      - publicRead
+      - publicReadWrite
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Default access controls for objects in the bucket
+      isArray: false
+      name: default_object_acl
+      predefined:
+      - authenticatedRead
+      - bucketOwnerFullControl
+      - bucketOwnerRead
+      - private
+      - projectPrivate
+      - publicRead
+      required: false
+      secret: false
+    deprecated: false
+    description: Creates a new bucket
+    execution: false
+    name: gcs-create-bucket
+  - arguments:
+    - default: true
+      description: Name of the bucket to delete
+      isArray: false
+      name: bucket_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'False'
+      description: Force-delete the bucket if not empty
+      isArray: false
+      name: force
+      predefined:
+      - 'True'
```

Comment:
please use `true` and `false`

Reply:
It has to be strings (otherwise the IDE plugin breaks).